container/k8s/base_deploy.py

```diff
@@ -323,13 +346,10 @@ def _service_to_container(name, service):
                     if vols:
                         volumes += vols
```

Comment:
This can lead to duplicate volumes if two containers are mounting the same volume